front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Settings/index.tsx

```diff
@@ -3,12 +3,14 @@ import CustomPageSettingsForm from '../../CustomPageSettingsForm';
 import useCustomPage from 'hooks/useCustomPage';
 import { useParams } from 'react-router-dom';
 import { isNilOrError } from 'utils/helperUtils';
-import { updateCustomPage } from 'services/customPages';
+import { ICustomPageData, updateCustomPage } from 'services/customPages';
 import { FormValues } from 'containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm';
+import useNavbarItem from 'modules/commercial/customizable_navbar/hooks/useNavbarItem';
```

Comment:
It's the main issue of this PR now. How can we solve it?

1. Move (or duplicate?) the `useNavbarItem` hook into the core and import it from the module
2. Move the entire `customizable_navbar` module into the core as anyway we want to make it available for OS https://citizenlab.atlassian.net/browse/CL-1768
3. Render `CustomPageSettingsForm` with `nav_bar_item_title_multiloc` in an outlet

@lilkraftwerk What do you think is better? Maybe something else?

Reply:
I think 1, just copy the entire file into an appropriate place in our normal hooks folder, then update all the imports to point there instead of to the module. We can check with Brent when he's back if that's "legal" under the module rules but as far as I know it should be fine, it's OK to import from core -> modules but not modules -> core, so if it lives in the core and is imported elsewhere it should be chill

Reply:
First, I did it. But then I realized it could be simplified, so we don't need this hook here anymore.